app.js

```diff
@@ -47,13 +47,15 @@ socket.on('chat', (data) => {
     const date = new Date(data.date)
     const hour = date.getHours()
     const min = date.getMinutes()
+	console.log('\n')
```

Comment:
What is this extra end line for? 
(`console.log()` already produces a end line)